components/google-cloud/docs/source/google_cloud_pipeline_components.aiplatform.rst

```diff
@@ -5,6 +5,6 @@ Module contents
 ---------------
 
 .. automodule:: google_cloud_pipeline_components.aiplatform
-   :members: ImageDatasetCreateOp, TabularDatasetCreateOp, TextDatasetCreateOp, VideoDatasetCreateOp ,ImageDatasetExportDataOp, TabularDatasetExportDataOp, TextDatasetExportDataOp, VideoDatasetExportDataOp, ImageDatasetImportDataOp, TextDatasetImportDataOp, VideoDatasetImportDataOp, CustomContainerTrainingJobRunOp, CustomPythonPackageTrainingJobRunOp, AutoMLImageTrainingJobRunOp, AutoMLTextTrainingJobRunOp, AutoMLTabularTrainingJobRunOp, AutoMLVideoTrainingJobRunOp, ModelDeployOp, ModelBatchPredictOp, ModelUploadOp, EndpointCreateOp
+   :members: TimeSeriesDatasetCreateOp, TimeSeriesDatasetExportDataOp, AutoMLForecastingTrainingJobRunOp,ImageDatasetCreateOp, TabularDatasetCreateOp, TextDatasetCreateOp, VideoDatasetCreateOp ,ImageDatasetExportDataOp, TabularDatasetExportDataOp, TextDatasetExportDataOp, VideoDatasetExportDataOp, ImageDatasetImportDataOp, TextDatasetImportDataOp, VideoDatasetImportDataOp, CustomContainerTrainingJobRunOp, CustomPythonPackageTrainingJobRunOp, AutoMLImageTrainingJobRunOp, AutoMLTextTrainingJobRunOp, AutoMLTabularTrainingJobRunOp, AutoMLVideoTrainingJobRunOp, ModelDeployOp, ModelBatchPredictOp, ModelUploadOp, EndpointCreateOp
```

Comment:
I think this change should be added to master too, is that right?
How do you plan to manage this?

Reply:
Great catch! I have created a follow up PR #5953 to update master accordingly. Sincere appreciate your diligent review.